docs/Configuration.md

```diff
@@ -162,6 +162,27 @@ resolutions:
 For details of the available reasons, see
 [RuleViolationResolutionReason.kt](../model/src/main/kotlin/config/RuleViolationResolutionReason.kt)
 
+### Curations
+
+#### License Findings
+
+Scan result represent the detected licenses as collection of license findings. A single `LicenseFinding` is represented 
+as tuple: `(license-Id, file-path, start-line, end-line)`. A `LicenseFindingCuration` when applied changes the license-Id
```

Comment:
"Applying a `licenseFindingCuration` changes the..."

Reply:
Done.